content/en/docs/concepts/wasm/index.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Throughout this document, we're referring to extensions as "plugins", "extensions" and "filters". We really need to pick one and stick to it.

Reply:
Tried to clean that up.  PTAL